timesketch/views/sketch.py

```diff
@@ -363,7 +363,15 @@ def timelines(sketch_id):
                     searchindex=searchindex)
                 db_session.add(_timeline)
                 sketch.timelines.append(_timeline)
-        db_session.commit()
+                db_session.commit()
+
+                # If enabled, run sketch analyzers when timeline is added.
+                from timesketch.lib import tasks
```

Comment:
Why not import at the top of the file?  Maybe add a comment about that?

Reply:
Same comment as above. Added comment to explain as well.